packages/fluentui/react-northstar/src/components/Form/FormButton.tsx

```diff
@@ -3,7 +3,9 @@ import { commonPropTypes } from '../../utils';
 import { Button, ButtonProps } from '../Button/Button';
 import { _FormFieldBase, FormFieldBaseProps } from './utils/formFieldBase';
 
-interface FormButtonOwnProps extends ButtonProps {}
+interface FormButtonOwnProps extends ButtonProps {
+  type?: string;
```

Comment:
Interesting, I wonder why this isn't  available in `ButtonProps` it's a valid attr for the native `Button`